agent/engine/docker_task_engine.go

```diff
@@ -1244,6 +1244,8 @@ func (engine *DockerTaskEngine) provisionContainerResources(task *apitask.Task,
 	taskIP := result.IPs[0].Address.IP.String()
 	seelog.Infof("Task engine [%s]: associated with ip address '%s'", task.Arn, taskIP)
 	engine.state.AddTaskIPAddress(taskIP, task.Arn)
```

Comment:
is the plan to fully migrate to boltdb and then remove the state save here?

Reply:
yeah, i plan to remove these usages of state manager after boltdb changes are done